### PR TITLE
Adds unit tests

### LINKED ISSUE #56: Add unit tests
The tests where removed in #44, not having any tests makes it hard understand and refactor code.

### PR DESCRIPTION
**Pull request by sverhoeven**
Fixes #56

### CODE REVIEW

File: tests/deterministic/test_sine.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import numpy as np
+from numpy.testing import assert_almost_equal
+
+from sequgen.deterministic.sine import sine
+
+
+def test_with_defaults():
+    t_predict = np.arange(15)
```
Comment (eriktks) [inline tests/deterministic/test_sine.py:8]:
```suggestion
    t_predict = np.linspace(0, 1, 20)
```

File: tests/deterministic/test_triangular_peak.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+from sequgen.deterministic.triangular_peak import triangular_peak
+
+
+def test_with_defaults():
+    t_predict = np.arange(7)
+
+    result = triangular_peak(t_predict, width_base_right=2)
+
+    # TODO expected peak=1 at position 0
+    expected = np.array([0.5, 0., 0., 0., 0., 0., 0.])
```
Comment (eriktks) [inline tests/deterministic/test_triangular_peak.py:13]: These expected values are unexpected for me (reviewer)

**Review by eriktks — APPROVED** [review]
Great work!

**Comment by sverhoeven** [comment]
Thanks for reviewing

**Pull request merged**